scripts/prepare_sector_network.py

```diff
@@ -2170,7 +2170,7 @@ def add_fuel_cell_cars(n, p_set, fuel_cell_share, temperature):
         suffix=" land transport fuel cell",
         bus=spatial.h2.nodes,
         carrier="land transport fuel cell",
-        p_set=profile,
+        p_set=profile.loc[n.snapshots],
```

Comment:
Why is this ?

Reply:
Sometimes the snapshots and the profile data are not aligned. If you looked at `def add_EVs()` they did this snapshot filter already, so its just applying to other transport technologies